### DISCUSSION TITLE
Price Range Slider not working

### DISCUSSION THREAD

**Discussion by rutulritz** [breezefront/community#76] [1. Help] [answered]
Hi,

I refereed your link {https://github.com/swissup/range-slider} for Price range slider using java-script. It is working fine till some digit values. 

Reference : https://github.com/breezefront/community/discussions/29#discussioncomment-4225645

But in my website large digit values contained for that the slider is not showing selected ranges on initial state or on page load. 
![image](https://user-images.githubusercontent.com/9527439/226368647-236af7f0-9aec-4e26-bd81-1856b681a42c.png)

Below is the generated sliders :
![image](https://user-images.githubusercontent.com/9527439/226368195-4c2db9db-efe9-449e-8e51-3c757744f0b8.png)

Is there any configuration from javascript to implement the large digit price range slider?

Please help me on this.

Thanks

**Comment by vovayatsyuk** [comment, marked as answer]
Sorry, I don't understand what the issue is about. Here is a range slider example with large values: https://codepen.io/vovayatsyuk/pen/yLxxvpa

**Reply by rutulritz** [reply]
It is working fine. 

Thanks @vovayatsyuk

**Reply by rutulritz** [reply]
One more thing, Can you help me ? How can I init AJAX or Turbolink progress bar in custom URL calling or page load?

**Reply by vovayatsyuk** [reply]
You can use `Turbolinks.visit(url)` to navigate to the URL using turbo. View more info at https://github.com/turbolinks/turbolinks

**Reply by rutulritz** [reply]
Thanks a lot @vovayatsyuk
